### DISCUSSION TITLE
Remove special symbols in the title

### DISCUSSION THREAD

**Discussion by omdenis** [DominikPieper/obsidian-ReadItLater#89] [Ideas]
Hi,

I tried to use the plugin, but got some annoying issue. Some youtube authors put special symbols in their youtube title. So it's impossible to import their description.

Ex: : ? /

Maybe it could be removed from the string? As an option

PS. Only for the file name. Template I think could manage it.

Best Regards,
Denis

**Comment by adamluckdev** [comment]
Hi,

can you provide any link so I can test the issue? Also, please paste your templates (title and note) for YouTube videos.

Thanks

**Reply by ChrisM0815** [reply]
Hi,
I had this same issue today. I use the %title% variable in the filename which prevents some files, where the title contains certain symbols ( | . / \ # ^ [ ] ) from being created. 

When I use https://www.youtube.com/watch?v=rK3xcRYLWeg for example i get the following Error:

```
plugin:obsidian-read-it-later:59 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'querySelector')
    at Qt.parseSchema (plugin:obsidian-read-it-later:59:20126)
    at async Qt.prepareNote (plugin:obsidian-read-it-later:59:17409)
    at async wt.processClipboard (plugin:obsidian-read-it-later:146:2921)
    at async HTMLDivElement.eval (plugin:obsidian-read-it-later:146:2313)
parseSchema @ plugin:obsidian-read-it-later:59
```

My Template looks as follows:
`Youtube - %title%`
```yaml
---
up: 
tags: 
- video
- inbox
url: %videoURL%
channel: %channelName%
channelurl: %channelURL%
todo: false
---
%videoPlayer%
```
It works well with all Videos that do not contain any special characters in the title. It would be great to have an option to specify symbols that should be filtered out.
Best Regards,
Christoph
